### PR TITLE
Create server hosting guide for compute

### PR DESCRIPTION
**Pull request by aayushmahapatra**
- Go
- Rust
- Ruby on Rails
- Express

### CODE REVIEW

**Review by rekpero — APPROVED** [review]

File: .DS_Store
Comment (shomix) [inline .DS_Store:1]: can you remove all of the `.DS_Store` files?

File: examples/expressapp/app.js
Comment (shomix) [inline examples/expressapp/app.js:1]: docker file is missing for expressapp?

**Review by shomix — APPROVED** [review]
Some minor comments

File: .DS_Store
Comment (aayushmahapatra) [reply .DS_Store:1]: Done.

File: examples/expressapp/app.js
Comment (aayushmahapatra) [reply examples/expressapp/app.js:1]: Added.

**Review by rekpero — APPROVED** [review]

**Pull request merged**